lib/events/filesessions/fileasync.go

```diff
@@ -514,7 +521,7 @@ func (u *Uploader) upload(up *upload) error {
 		}
 	}
 
-	if err := stream.Complete(u.ctx); err != nil {
+	if err = stream.Complete(u.ctx); err != nil {
```

Comment:
Nit: this can be kept as `if err := ...`.  Variable bindings within `if` blocks are allowed to shadow variable bindings in the outer scope.  This is almost always preferable since it means that the code won't need to be changed in the future if the variables in the outer scope are changed down the road.